test/runtests.jl

```diff
@@ -4,7 +4,11 @@ using Compat.LinearAlgebra
 using Rotations
 using StaticArrays
 
-import Compat.Random: srand
+if VERSION < v"0.7.0-beta2.171"
+    import Compat.Random: srand
```

Comment:
General `Compat` style is to use the newer API, i.e. `import Compat.Random: seed!`